src/SDK/API.php

```diff
@@ -20,7 +20,7 @@ class API
     /**
      * Constructor for API class.
      *
-     * @param string $projectId
+     * @param string      $projectId
```

Comment:
tab?

Reply:
also in other places. Is this part of the syntax or linter?

Reply:
worth using a formatter for everything